### DISCUSSION TITLE
HD300 remote - looking for one

### DISCUSSION THREAD

**Discussion by CptKirk-1701** [OpenSageTV/Community#177] [General]
Anyone have one not being used anymore?

Will pay for shipping and unit cost.

Let me know.

**Comment by LoisGNS** [comment]
I don't have a remote to pass along, however, I recently bought a Sofabaton U1 universal remote for a little over $20 (on sale). It has a large database of supported devices, plus the ability to learn codes from an existing remote. I was planning to use the learn-mode to get to talk to Sage, but it turns out that Sage was already in its database, so I how have another usable remote. It supports multiple devices, and I was also able to get it to control an old VCR-DVD device for which I never had the remote. You have to connect it to an app to set things up, but you can also change the commands attached to keys and even assign macros to keys. I've only started exploring what I can do with it, but it's proving useful so far and worth the price.

**Reply by mickgeller** [reply]
Another option is to find an old Logitech Harmony remote or a friend that will give you one. Do not buy a new one. Prices have sky rocketed.
Then go to buttonworx.com, buy a rebuild kit, and turn it into a new one, following the directions there. Hint 90% isopropyl alcohol is a winner to clean it.

I cannot speak for the software to program it, as I already had that when I rebuilt two of them. My guess is that the "MyHarmony" software is available somewhere.

**Comment by phantomfsoc** [comment]
I have a bunch of Sage Remotes at home. I can check if there is one if you still need. 
I'm in Toronto Canada.

**Comment by Sagetvmaster** [comment]
> Anyone have one not being used anymore?
> 
> Will pay for shipping and unit cost.
> 
> Let me know.

Is there a difference between a HD300 and HD200 remote, as far as your needs go?
